hudi-common/src/main/java/org/apache/hudi/common/util/ClusteringUtils.java

```diff
@@ -111,7 +112,7 @@ public static Option<HoodieInstant> getInflightClusteringInstant(String timestam
   public static Option<HoodieInstant> getRequestedClusteringInstant(String timestamp, HoodieActiveTimeline activeTimeline, InstantGenerator factory) {
     HoodieTimeline pendingReplaceOrClusterTimeline = activeTimeline.filterPendingReplaceOrClusteringTimeline();
     HoodieInstant requestedInstant = factory.getClusteringCommitRequestedInstant(timestamp);
-    if (pendingReplaceOrClusterTimeline.containsInstant(requestedInstant)) {
+    if (pendingReplaceOrClusterTimeline.containsInstant(timestamp)) {
```

Comment:
why change this? An inflight instant also works here?

Reply:
we are justing using the overloaded method which takes in string (timestamp) rather than the HoodieInstant.